src/alert-manager/src/cluster-utilization/send_alert.py

```diff
@@ -44,131 +58,154 @@ def get_job_usage_info(job_usage_result, rest_url):
     job_list = resp.json()
 
     for v in job_usage_result["data"]["result"]:
-        job_infos[v["metric"]["job_name"]] = {"usage": v["value"][1][:6] + "%"}
+        job_infos[v["metric"]["job_name"]] = {
+            "job_name": v["metric"]["job_name"],
+            "usage": v["value"][1][:6] + "%"
+        }
+    for v in user_usage_result["data"]["result"]:
+        user_infos[v["metric"]["username"]] = {
+            "username": v["metric"]["username"],
+            "usage": v["value"][1][:6] + "%", "resources_occupied": 0
+        }
     for job_name, job_info in job_infos.items():
         url = urllib.parse.urljoin(rest_url + "/", job_name)
         resp = requests.get(url, headers=headers)
         if not resp.ok:
             logging.warning("request failed %s", resp.text)
+            del job_infos[job_name]
```

Comment:
Please make sure this operation is safe. You delete element from `job_infos` while iterate it. I am not sure the behavior in this case.

Reply:
> Please make sure this operation is safe. You delete element from `job_infos` while iterate it. I am not sure the behavior in this case.

Fixed & thanks for the reminder, there will be an issue in python3 if we delete elements while iterating.